doc/source/user_guide/make_container.rst

```diff
@@ -0,0 +1,106 @@
+.. _ref_make_container:
+
+Create your Fluent Docker container
```

Comment:
1. This heading will be rendered with the same font size as _Requirements_ etc. 
Is that intentional? Aren't those subsections?
2. I don't think that headings should contain the word _your_.
3. The guidance is for creating a container _image_.

Reply:
Done. Thank you.